package com.revature.alchemyapp.exceptions;

public class FailedAuthenticationException extends Exception {

}
